Conversion.js

```diff
@@ -1,3 +1,5 @@
 var initial = measure('1 cup');
 var final = initial.add('1 tbsp').format('d Mm')
 console.log(final); // "1.0625 Cup"
+
+// Test.
```

Comment:
This isn't bad, but we should not commit these types of comments. Instead it's better to restrict to summary and documentation.